README.md

```diff
@@ -52,7 +52,47 @@ Generate team
 
 #### Configuring GatorGrade Checks
 
-Input team
+There are multiple customizable options with gatorgrade!
+Gatorgrader checks are able to run within a specific file context
```

Comment:
Since this section is about how to _configure_ GatorGrader checks, and not about how to _run_ GatorGrader checks, might it be clearer to replace all of the uses of the word "run" with phrases like "be configured" or "configure"?
For example, this sentence could be `GatorGrader checks are able to be configured to run for a specific file or without any file path.`